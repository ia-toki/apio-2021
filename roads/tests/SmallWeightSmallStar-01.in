c10234cabdfd6eae47773fb6f163e0350fc797e1
188
0 73 7
0 161 7
0 172 10
0 88 8
0 180 9
0 81 7
0 28 8
0 92 8
0 57 7
0 66 10
0 108 8
0 156 10
0 142 8
0 65 6
0 130 6
0 166 6
0 132 7
0 43 7
0 87 7
0 171 8
0 53 5
0 151 9
0 68 5
0 37 8
0 67 9
0 99 5
0 15 5
0 148 10
0 31 7
0 46 8
0 23 5
0 162 5
0 168 6
0 136 5
0 1 9
0 109 9
0 29 6
0 24 10
0 117 9
0 157 5
0 39 9
0 20 8
0 102 7
0 179 7
0 97 10
0 115 5
0 165 5
0 11 9
0 116 9
0 110 8
0 164 6
0 134 5
0 35 9
0 3 5
0 121 10
0 112 10
0 128 7
0 96 9
0 9 5
0 150 5
0 185 8
0 62 8
0 126 6
0 78 7
0 93 5
0 163 5
0 71 5
0 186 5
0 58 9
0 10 9
0 2 7
0 51 6
0 76 6
0 174 5
0 144 9
0 167 10
0 32 8
0 119 6
0 17 5
0 64 6
0 100 10
0 72 6
0 74 9
0 114 10
0 85 7
0 95 10
0 80 6
0 86 7
0 133 5
0 153 7
0 124 9
0 91 7
0 41 7
0 138 6
0 47 10
0 181 10
0 106 9
0 77 6
0 54 6
0 158 6
0 173 9
0 127 9
0 98 8
0 44 7
0 135 6
0 141 7
0 45 5
0 83 8
0 63 10
0 149 9
0 52 6
0 137 5
0 75 7
0 27 8
0 177 6
0 120 7
0 6 6
0 89 7
0 113 6
0 90 8
0 79 9
0 184 10
0 14 7
0 125 9
0 61 7
0 69 7
0 145 8
0 155 6
0 40 5
0 140 6
0 38 6
0 170 9
0 152 9
0 13 10
0 129 5
0 101 8
0 22 9
0 50 5
0 169 5
0 55 10
0 182 5
0 122 8
0 42 9
0 143 5
0 30 8
0 131 5
0 21 6
0 12 10
0 26 6
0 18 5
0 34 10
0 147 10
0 5 10
0 56 8
0 104 8
0 178 6
0 154 7
0 105 7
0 107 5
0 118 6
0 111 9
0 7 10
0 82 9
0 159 7
0 103 6
0 8 9
0 60 9
0 4 7
0 176 7
0 33 6
0 84 8
0 48 9
0 70 8
0 139 7
0 49 7
0 160 5
0 183 7
0 36 10
0 146 5
0 187 9
0 19 9
0 123 10
0 175 9
0 94 6
0 25 5
0 59 6
0 16 9
