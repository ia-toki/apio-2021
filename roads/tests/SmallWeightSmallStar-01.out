064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
1376 1366 1356 1346 1336 1326 1316 1306 1296 1286 1276 1266 1256 1246 1236 1226 1216 1206 1196 1186 1176 1166 1156 1146 1136 1126 1117 1108 1099 1090 1081 1072 1063 1054 1045 1036 1027 1018 1009 1000 991 982 973 964 955 946 937 928 919 910 901 892 883 874 865 856 847 838 829 820 811 803 795 787 779 771 763 755 747 739 731 723 715 707 699 691 683 675 667 659 651 643 635 627 619 611 604 597 590 583 576 569 562 555 548 541 534 527 520 513 506 499 492 485 478 471 464 457 450 443 436 429 422 415 408 401 394 387 380 373 367 361 355 349 343 337 331 325 319 313 307 301 295 289 283 277 271 265 259 253 247 241 235 229 223 217 211 205 199 193 187 181 175 170 165 160 155 150 145 140 135 130 125 120 115 110 105 100 95 90 85 80 75 70 65 60 55 50 45 40 35 30 25 20 15 10 5 0
